content-src/asrouter/components/ModalOverlay/ModalOverlay.jsx

```diff
@@ -15,11 +15,21 @@ export class ModalOverlayWrapper extends React.PureComponent {
   componentWillMount() {
     this.props.document.addEventListener("keydown", this.onKeyDown);
     this.props.document.body.classList.add("modal-open");
+
+    this.props.document
+      .getElementById("header-asrouter-container")
```

Comment:
Can move getElementById("header-asrouter-container") in constructor as this.header = this.props.document.getElementById("header-asrouter-container"); and use it to set and unset css

Reply:
In order to fix the tests, I think the right thing to do here is to check the existence of the element before adding the class, i.e.
``` js
this.header = this.props.document.getElementById("header-asrouter-container"); 

...
if (this.header) {
  ...
}

```

Reply:
The other thing you could do is pass `props.document` further down the chain, but because some of this code is being removed soon i don't think it's worth it.

Reply:
(The reason the tests are failing is because `props.document` is not passed down to `ModalOverlayWrapper` from the outer asrouter-content, so it's just using the default `global.document` instead)